pkg/envoy/embedded_envoy.go

```diff
@@ -248,9 +248,10 @@ func newEnvoyLogPiper() io.WriteCloser {
 			case "off", "critical", "error":
 				scopedLog.Error(msg)
 			case "warning":
-				// Silently drop expected warnings if flowdebug is not enabled
-				// TODO: Remove this special case when https://github.com/envoyproxy/envoy/issues/13504 is fixed.
```

Comment:
looks like the related envoy issue is still not fixed (at least not closed).

if this is still relevant - we should keep this exception?

Reply:
I read the discussion on the issue, it seems some aspects are fixed, and others not, hence the issue is not closed yet. I could not get the warning appear in my local testing, so it _looks like_ it was fixed _for our use cases_.

Reply:
answered in the commit message